src/components/blog/blogs/index.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react';
+import { Box, Container, Button, Grid } from '@mui/material';
+import { makeStyles } from '@mui/styles';
+import { LoadingButton } from '@mui/lab';
+
+import BlogsData from 'src/components/blog/blogs/blogs.json';
+import BlogItem from 'src/components/blog/blogs/BlogItem';
+import TextBadge from 'src/components/common/TextBadge';
+
+const useStyles = makeStyles({
+  BlogBannerContainer: {
+    padding: '3rem 0',
+    borderRadius: '15px',
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+    gap: '2rem',
+    '& > button:hover': {
+      backgroundColor: 'transparent !important',
+    },
+  },
+  loadMoreBtn: {
+    color: 'black',
+  },
+});
+
+export function BlogList() {
+  const classes = useStyles();
+  const [loading, setLoading] = useState(false);
+  const [blogsToShow, setBlogsToShow] = useState(9);
+  const handleLoadMore = () => {
```

Comment:
@UzeeChamp1 when you start writing functions in component, make sure to add empty line before them
```suggestion

  const handleLoadMore = () => {
```